blackbox/build.gradle

```diff
@@ -12,79 +12,118 @@ task bootstrap(type: Exec) {
     commandLine 'sh', "$projectDir/bootstrap.sh"
 }
 
-task unpackDistTar(dependsOn: [project(':app').installDist]) {
+task deleteCrateDist(type: Delete) {
+    delete crateDist
+}
+
+task unpackDistTar(dependsOn: [project(':app').installDist, deleteCrateDist  ]) {
+    println("Copy distribution from " + (project(':app').installCommunityEditionDist.destinationDir) + " to " + crateDist)
     doLast {
         copy {
             from(project(':app').installDist.destinationDir) {
                 includeEmptyDirs = true
             }
             into crateDist
         }
+        lessLogging()
+        ignoreDiskThreshold()
     }
     outputs.dir crateDist
 }
 
-task lessLogging(dependsOn: unpackDistTar) {
+task unpackCommunityEditionDistTar(dependsOn: [project(':app').installCommunityEditionDist, deleteCrateDist]) {
+    println("Copy distribution from " + (project(':app').installCommunityEditionDist.destinationDir) + " to " + crateDist)
     doLast {
-        def file = new File("$projectDir/tmp/crate/config/log4j2.properties")
-        file.write(file.text.replaceAll('rootLogger.level = info', 'rootLogger.level: warn'))
+        copy {
+            from(project(':app').installCommunityEditionDist.destinationDir) {
+                includeEmptyDirs = true
+            }
+            into crateDist
+        }
+        lessLogging()
+        ignoreDiskThreshold()
     }
+    outputs.dir crateDist
 }
 
-task ignoreDiskThreshold(dependsOn: unpackDistTar) {
-    doLast {
-        def file = new File("$projectDir/tmp/crate/config/crate.yml")
-        file.write(file.text.replaceAll(
-                '# cluster.routing.allocation.disk.threshold_enabled: true',
-                'cluster.routing.allocation.disk.threshold_enabled: false')
-        )
-    }
+def lessLogging() {
+    def file = new File("$projectDir/tmp/crate/config/log4j2.properties")
+    file.write(file.text.replaceAll('rootLogger.level = info', 'rootLogger.level: warn'))
 }
 
-task hdfsTest(type: Exec) {
+def ignoreDiskThreshold() {
+    def file = new File("$projectDir/tmp/crate/config/crate.yml")
+    file.write(file.text.replaceAll(
+        '# cluster.routing.allocation.disk.threshold_enabled: true',
+        'cluster.routing.allocation.disk.threshold_enabled: false')
+    )
+}
+
+task buildDocs(type: Exec, dependsOn: bootstrap) {
+    commandLine "$projectDir/bin/sphinx"
+}
+
+task developDocs(type: Exec, dependsOn: bootstrap) {
+    commandLine "$projectDir/bin/sphinx", "dev"
+}
+
+task hdfsTest(type: Exec,
+    dependsOn: ['bootstrap', 'unpackDistTar', ':es:es-repository-hdfs:jar',
+                project(':es:es-repository-hdfs').blackBoxTestJar, ':enterprise:functions:jar'
```

Comment:
why does it now depends on `:enterprise:functions:jar`? I think the test has no dependency to that
also `':es:es-repository-hdfs:jar'`  dependency is new.